src/test/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACLTest.java

```diff
@@ -61,13 +62,7 @@ of this software and associated documentation files (the "Software"), to deal
 import org.powermock.modules.junit4.PowerMockRunner;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Wildcard imports are a bad practice which make it hard to see where classes are coming from.  Please do not use wildcard imports.

Reply:
Okay. Will fix it.

Reply:
Fixed the wildcard imports.